sqld/src/error.rs

```diff
@@ -29,6 +29,8 @@ pub enum Error {
     ReplicatorExited,
     #[error("Timed out while openning database connection")]
     DbCreateTimeout,
+    #[error("Response payload exceeds the maximum allowed size, consider limiting the retreived results.")]
```

Comment:
Some things about this message:

1. Since we're not calculating the size exactly, we should be more clear about that.  It's actually the *estimated* size that we're limiting, based on the number of rows (and the estimated size of each row?).
2. The maximum size itself isn't stated. Could we give some indicator here?
3. It would be great if this message could be formatted to indicate the number of actual rows that would theoretically be allowed to pass for the query that failed. It looks like we know that from `execute_query_inner`?  Without some information here, the reader of this message doesn't really know where to start truncating, and would have to experimentally bisect until the query works.